tests/unit/utils/highcharts/spline-chart-test.js

```diff
@@ -84,9 +88,6 @@ module('Unit | Utility | highcharts/spline-chart', function () {
 
           tooltip: {
             crosshairs: true,
-            formatter: function tooltip() {
-              return `${this.point}`;
-            },
```

Comment:
I imagine that you might have had to pass the same function in order for `deepEqual` to pass, similarly to how, in JavaScript, `{} === {}` is not equal to `true` because the objects refer to a different instance.

```javascript
const formatterFunction = ...;

// Setup
tooltip: formatterFunction

// Assertion
formatter: formatterFunction
```

Let's avoid checking the formatter function by deleting it from `options`.